tests/admin_changelist/tests.py

```diff
@@ -445,6 +445,81 @@ def test_distinct_for_many_to_many_at_second_level_in_search_fields(self):
         # There's only one Concert instance
         self.assertEqual(cl.queryset.count(), 1)
 
+    def test_no_distinct_for_m2m_in_list_filter_without_lookup(self):
+        """
+        Regression test for #18729: When having a ManyToMany in list_filter (Basic ManyToMany),
```

Comment:
"Regression test for #18729:" isn't needed since the comments describe the issue (see the test docstring advice in [Python style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)).

Also, perhaps these tests could be condensed into a loop since they're so similar? That said, do we even need so many similar tests? It doesn't seem like the different cases are testing different branches since only I place was changed.